### PR TITLE
Update client readme

### PR DESCRIPTION
**Pull request by zeibura**
Signed-off-by: Zeibura Kathau <zeibura.kathau@broadcom.com>

Update client readme for new release

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/README.md [hunk 1]
```diff
@@ -30,29 +30,68 @@ Autocomplete speeds up the coding process by intuitively suggesting the most lik
 - COBOL keywords
 - COBOL variables
 - COBOL paragraphs
-- Code Snippet
-- Copybook variables
-- Copybook paragraphs
+- CICS keywords
+- Code Snippets
+- Copybook variable and paragraph names
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/README.md:35]: Also sections and subroutines

File: clients/cobol-lsp-vscode-extension/README.md [hunk 2]
```diff
@@ -93,28 +134,60 @@ Copybooks downloaded from the mainframe using an older version of COBOL Language
...
+
+Syntax and semantic analysis are enabled for keywords, variables, and paragraphs across all copybooks used in the COBOL file, to ensure and maintain compatibility of copybooks called in code.
+
+The semantic analysis feature takes into account `COPY REPLACING` statements which alter the content of copybooks when checking for errors.
+
+![Semantic check before and after a REPLACING statement](/docs/images/CPYReplacing.gif)
+
+#### Syntax Coloring
+
+Syntax coloring is automatically enabled for copybook files with the extension `.cpy`, as long as they are used in the main COBOL file.
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/README.md:147]: I'm not sure, but I think we should explain in more detail that the user may enable the coloring for copybooks without extensions by selecting "COBOL Copybook" language id

**Review by zimlu02 — COMMENTED** [review]
Isn't this readme supposed to be exactly the same as the from the root?

File: clients/cobol-lsp-vscode-extension/README.md (hunk 1 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/README.md:35]: Subroutines are mentioned

File: clients/cobol-lsp-vscode-extension/README.md [hunk 3]
```diff
@@ -30,29 +30,68 @@ Autocomplete speeds up the coding process by intuitively suggesting the most lik
...
+
+![Code snippets](/docs/images/Snippet.gif)
+
+You can also insert a code snippet by typing the name of the snippet in your code and clicking on the autocomplete text.
+
+The COBOL Language Support extension also supports user snippets. Add your custom snippets to the `COBOL.json` file in your user snippets folder.
+
+## Subroutine Support
+
+The COBOL Language Support extension supports subroutines specified in CALL statements if the called program is stored in a local folder in your workspace. The Go To Definition and Find All References functionalities, as well as autocomplete, are extended to work for names of subroutines.
```
[the remark this answers is not in the document]
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/README.md:76]: how is it redundant, the first "support" is part of the name of the extension

File: clients/cobol-lsp-vscode-extension/README.md (hunk 2 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/README.md]: Done (on both readmes)

**Comment by zeibura** [comment]
This PR was just to get the client readme the same as the main one, yes. I've made Andrei's suggested change about the language mode on both.

File: clients/cobol-lsp-vscode-extension/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/README.md:76]: I just meant to use some other word not to repeat "support" into one sentence, but I have not found an example though. It is up to you, I don't insist

**Review by temanbrcom — APPROVED** [review]
🍰

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**